client/src/pages/home.js

```diff
@@ -28,12 +28,6 @@ export default withStyles({
         <div class={`${classes.row} row u-center`}>
           <img src={config.logoUrl} class={`avatar ${classes.logo}`} />
         </div>
-        <div class={`${classes.row} row u-center`}>
-          <p>
-            Infrastructure sponsored by
-            <a href='https://g.co/cloud'>g.co/cloud</a>
```

Comment:
I think this would be hard because we would need to allow for HTML code injection, which I'm not sure if I'm entirely comfortable with? We could probably; if they control config they probably control client-side rendering?